projects/fhi-angular-components/src/styles/fhi-ng-selects/fhi-ng-select-wrapper.scss

```diff
@@ -40,7 +30,8 @@
                 }
 
                 .ng-placeholder {
-                    color: $fhi-core-grey-3; // TODO: need $fhi-input-placeholder-color in core.variables and $input-placeholder-color: $fhi-input-placeholder-color
+                    color: $fhi-core-grey-3;
+                    z-index: 3 !important;
```

Comment:
Avoid using `!important` as it breaks css priority. Try using more spesific selectors instead.

Reply:
... got lazy since I'm working on a dead product, but you're right, we should be disciplined to the bitter end 😉